front/lib/api/assistant/pubsub.ts

```diff
@@ -120,7 +120,7 @@ export async function* getConversationEvents(
       const events = await redis.xRead(
         { key: pubsubChannel, id: lastEventId ? lastEventId : "0-0" },
         // weird, xread does not return on new message when count is = 1. Anything over 1 works.
-        { COUNT: 1, BLOCK: 60 * 1000 }
+        { COUNT: 100, BLOCK: 60 * 1000 }
```

Comment:
What does it do ?

Reply:
Why 100?

Reply:
When reading from the stream in blocking mode (XREAD block 60secs) two things are possible:
- there is no new event > than lastEventId, so block for timeout until you get one event, and answer with that event
- there are already events in the stream with ids < lastEventId, in which case, return up to 100.

100 seems like a reasonable default to get the full stream in one XREAD call in (I would assume) 99% of the time.

Let me know if you think otherwise, I am not married to any value here. But something too small like one would make us do a large amount of unnecessary network round trips all the time as we would read the existing event stream one event by one event.

Reply:
Let’s use a power of 2 a bit lower, say 32

Reply:
sounds good.